### ISSUE TITLE
make HB JWT mandatory

### ISSUE THREAD

**Issue by Sakurann** [vcstuff/draft-terbu-sd-jwt-vc#110]
I thought we have previously agreed to make HB JWT mandatory ([current text reads MAY](https://vcstuff.github.io/draft-terbu-sd-jwt-vc/draft-terbu-sd-jwt-vc.html#section-6.1-2)). I think it is critical for the interoperability to mandate HB JWT. 

and with [PR 262](https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/262) in sd-jwt being merged, SD-JWT will normatively define the structure of HB-JWT so section 6.1.1 in sd-jwt vc should also be removed.

**Comment by paulbastian** [comment]
I agree moving the HB-JWT to SD-JWT spec and removing it from SD-JWT-VC but I would like to keep cryptographic holder binding optional. Claim based holder binding is a simple and easy solution that will be requested by adopters.

**Comment by paulbastian** [comment]
At the same time it might make sense to give more guidance in this direction in the spec

**Comment by tlodderstedt** [comment]
I think the vc-sd-jwt should be the module describing how VCs shall be implemented with SD-JWT, however it should be flexible enough to be used in different contexts. 

 Interoperability is the point of the high assurance profile. It does already make the HB JWT mandatory. I think this is sufficient.

**Comment by awoie** [comment]
@Sakurann I agree with @tlodderstedt that key binding should stay optional in this specification. More specific profiles may or may not make it mandatory. 

I'm marking as pending close and will close after 1 week of no activity.
